postgres-appliance/postgres_backup.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
```

Comment:
Different commit: could be, I'll have to squash some commits then, different PR: No, as the previous script (postgres_ha.sh) is split into postgres_ha.py and postgres_backup.sh (and cron).